opendebates/tests/test_views.py

```diff
@@ -23,3 +26,21 @@ def test_list_ideas_uses_site_mode_from_context(self, gsm_mock):
         """
         self.client.get(self.url)
         self.assertEqual(gsm_mock.call_count, 0)
+
+    def test_top_30_not_visible_if_no_previous_debate(self):
+        mode, _ = SiteMode.objects.get_or_create()
+        mode.previous_debate_time = None
+        mode.save()
+
+        response = self.client.get(self.url)
+        self.assertNotContains(response, "Top 30 Since Last Debate")
+
+    def test_top_30_visible_if_previous_debate(self):
+        mode, _ = SiteMode.objects.get_or_create()
+        mode.previous_debate_time = timezone.make_aware(
+            datetime.datetime(2016, 1, 1, 12)
+        )
+        mode.save()
+
+        response = self.client.get(self.url)
+        self.assertContains(response, "Top 30 Since Last Debate")
```

Comment:
Hmm.  Is top 30 visible if previous debate is non-null but in the future?

I *think* ideally it wouldn't be -- that way moderators could actually set it ahead of time and forget about it, relying on the system to start incrementing and make the sort option visible only after the right time.